e2e/tests/session.feature

```diff
@@ -2,12 +2,20 @@ Feature: Session Namespace
 
   Scenario: Supplier completes claim/proof lifecycle for a valid session
     Given the user has the pocketd binary installed
-    When the supplier "supplier1" has serviced a session with "5" relays for service "svc1" for application "app1"
+    # The number of relays serviced is set to make the resulting compute units sum
+    # above the current ProofRequirementThreshold governance parameter so a proof
+    # is always required.
+    # TODO_TECHDEBT(#745): Once the SMST is updated with the proper weights,
+    # using the appropriate compute units per relay, we can then restore the
+    # previous relay count.
+    When the supplier "supplier1" has serviced a session with "21" relays for service "anvil" for application "app1"
     And the user should wait for the "proof" module "CreateClaim" Message to be submitted
     And the user should wait for the "proof" module "ClaimCreated" tx event to be broadcast
-    Then the claim created by supplier "supplier1" for service "svc1" for application "app1" should be persisted on-chain
```

Comment:
I think this is coming from the assertion we make about the claims count
```
	require.Greater(s, len(allClaimsRes.Claims), len(preExistingClaims), "number of claims must have increased")
```

Which in some circumstances (tests repeated to quickly) fails because the previous claim gets removed.

I'll put back the step even if it's at the cost of some flakiness.